frame/balances/src/lib.rs

```diff
@@ -424,7 +425,9 @@ pub mod pallet {
 		Blake2_128Concat,
 		T::AccountId,
 		AccountData<T::Balance>,
-		ValueQuery
+		ValueQuery,
+		GetDefault,
+		ConstU32<300_000>,
```

Comment:
I guess this will need to be configurable in the future, because chains can have very different number of account